bin/GameIndex.dbf

```diff
@@ -1165,6 +1165,12 @@ Serial = SCED-51540
 Name   = Official PlayStation 2 Magazine Demo 39
 Region = PAL-M5
 ---------------------------------------------
+Serial = SCED-51531
```

Comment:
Are you sure this isn't a duplicate?

Reply:
It should fix the Primal demo on there but i left it commented because i don't have the game for testing.

Reply:
The serial is a duplicate.

Reply:
Oh you are right. I am completely wrong on this patch
SCED-51540
SCED-52037

Reply:
SCED-51568 is used for the Demo 33 which has primal